packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_counter/blocs/counter_bloc.dart

```diff
@@ -7,6 +7,7 @@
 
 import 'package:rx_bloc/rx_bloc.dart';
 import 'package:rxdart/rxdart.dart';
+import 'package:test_app/base/models/count.dart';
```

Comment:
Make sure that the path to the file is relative. This way we don't need to specify the project name in the path.